src/hooks/calculate_time.ts

```diff
@@ -0,0 +1,21 @@
+export function calculate_time(dateString: string | undefined) {
```

Comment:
JS 에서 sneak_case 는 지양합시다!
camelCase 로 변경 고고

Reply:
그리고 알고있겠지만 string | undefined 는 ?: 로도 가능합니당
```suggestion
export function calculateTime(dateString?: string) {
```

Reply:
근데 calculateTime 인데.. dateString 이 없는게 말이 되나..?

Reply:
어????....그...렇네요.....???!!!!!!

Reply:
undefinded일 때 처리 안하면 에러나는데 이유를 정확히 모르겠어요 ,,,,,,, 찾아보겠습니다.....!!!!!